README.md

```diff
@@ -27,21 +27,21 @@ in the *AWS Command Line Interface User Guide*.
 The credentials must have a policy applied that
 [allows access to Amazon ECR](http://docs.aws.amazon.com/AmazonECR/latest/userguide/ecr_managed_policies.html).
 
-## Building and Setup
-To build the Amazon ECR Docker Credential Helper, you must have Go 1.5 or greater, and you must have `git`
-and `make` installed on your system.
+## Installing
 
-Clone this repository into your existing `GOPATH` under
-`src/github.com/awslabs/amazon-ecr-credential-helper`, then run `make`.  The
-resulting binary can be found in `bin/local/docker-credential-ecr-login`.
+To build and install the Amazon ECR Docker Credential Helper, we suggest golang 1.7+ and `git` and `make` installed on your system.
```

Comment:
Any reason you picked 1.7+ instead of just 1.6?

Also: can you replace "golang" with "Go"?